### ISSUE TITLE
ELK Conversion results inconsistent with original in ELK Editor elkt/json

### ISSUE THREAD

**Issue by andrewj31415** [kieler/elk-live#63]
When I take the default starter example of http://rtsys.informatik.uni-kiel.de/elklive/elkgraph.html --that is,
```
algorithm: layered

node n1
node n2
node n3
edge n1 -> n2
edge n1 -> n3
```
paste it into the converter at http://rtsys.informatik.uni-kiel.de/elklive/conversion.html, and translate from `elkt` to `json`, the result is:
```
{
  "id": "n0",
  "children": [
    {
      "id": "n1"
    },
    {
      "id": "n2"
    },
    {
      "id": "n3"
    }
  ],
  "layoutOptions": {
    "algorithm": "layered"
  },
  "edges": [
    {
      "id": "e0",
      "sources": [
        "n1"
      ],
      "targets": [
        "n2"
      ]
    },
    {
      "id": "e1",
      "sources": [
        "n1"
      ],
      "targets": [
        "n3"
      ]
    }
  ]
}
```
which, when I paste it into http://rtsys.informatik.uni-kiel.de/elklive/json.html, gives:
http://rtsys.informatik.uni-kiel.de/elklive/json.html?compressedContent=N4KABGBECWAmkC4oDsAMkA04oGMAW0ANrAE4CmyiYA2thKBI1HFZMgIyR1gC+WTDJs3hI2AJi5M+3QUxgiUAZkmMe2ALr8ohAIYBPAPYBXAC4B5AA4noB5AGcqsqDsIBzAyWgm8AW1a69MnJ4bGkISDJYVzIHJFoBbnCWUTJ0LTk7YxIcGKp4oXCOFSZNRKgTHRJok1iaMsKJMvVuMMYnJIUIznTGSEyjbNy4+pROJp7wiqqyGryRtmUmlo0QHiA
which does not match the original--the nodes appear to have size 0.

Do you see the same behavior? Presumably this behavior is not intended.

**Comment by Eddykasp** [comment]
The conversion should be a simple conversion with no assumptions. This means we do not want to add fields to the json graph that are not explicitly present in the elkt version. Otherwise when we convert back to elkt they would then be added to the textual source, which would then be different than the original.

The elkt format specifically employs certain default values to make it easier to work with for humans whereas the json representation is a no-assumptions machine-readable format.

tl;dr convert(convert(elkt)) = elkt must always hold
